crates/bevy_render/src/camera/projection.rs

```diff
@@ -255,11 +255,14 @@ pub struct OrthographicProjection {
     ///
     /// Defaults to `ScalingMode::WindowSize(1.0)`
     pub scaling_mode: ScalingMode,
-    /// Scales the projection in world units.
+    /// Scales the projection.
     ///
     /// As scale increases, the apparent size of objects decreases, and vice versa.
     ///
-    /// Defaults to `1.0`
+    /// Note scaling can be set by [`scaling_mode`](Self::scaling_mode) as well.
```

Comment:
I think this looks better.
```suggestion
    /// Note: Scaling can be set by [`scaling_mode`](Self::scaling_mode) as well.
```

Reply:
I believe it should be "Note: scaling", not "Node: Scaling". Applied the rest, thanks!